Dockerize local development (#115)
onflow/kitty-items: README.md

```sh
git clone https://github.com/onflow/kitty-items.git
```
### 3. Install dependencies

- Run `npm install` in the root of the project.
- Run `npx lerna exec npm install` to install project dependencies.

### 4. Start the project

## Local development

- Run `./srcipts/run-local.sh` 

>   - This script exports the necessary configuration and starts the `web` and `api` applications in local Docker containers. (See`docker-compose.yml` for details.)

Reviewer: ```suggestion
  - This script exports the necessary configuration and starts the `web` and `api` applications in local Docker containers (see [docker-compose.yml](docker-compose.yml) for details).
```

>   - Local development uses the [Flow Emulator]() and the [FCL Dev-Wallet]() to simulate the blockchain and FCL compatible wallets.

Reviewer: ```suggestion
  - Local development uses the [Flow Emulator](https://docs.onflow.org/emulator/) and the [FCL Development Wallet](https://github.com/onflow/fcl-dev-wallet) to simulate the blockchain and an FCL-compatible wallet.
```

>   - All contracts are deployed to the Flow emulator.

Reviewer: ```suggestion
  - All contracts are deployed to the emulator.
```

## Testnet development

> #### Create a Flow Testnet account 

Reviewer: ```suggestion
### Create a Flow Testnet account 
```
